docs/root/intro/arch_overview/http/http_routing.rst

```diff
@@ -122,8 +122,9 @@ headers <config_http_filters_router_headers_consumed>`. The following configurat
   :ref:`retry priority <envoy_api_field_route.RetryPolicy.retry_priority>` can be
   configured to adjust the priority load used when selecting a priority for retries.
 
-Note that retries may be disabled depending on the contents of the :ref:`x-envoy-overloaded
-<config_http_filters_router_x-envoy-overloaded_consumed>`.
+Note that Envoy retries requests when :ref:`x-envoy-overloaded
+<config_http_filters_router_x-envoy-overloaded_set>` is present. It is recommended to configure
```

Comment:
This is already set to a default value of 3, so folks should already be protected. I also would mention retry budgets here instead of the max_retries CB threshold. Especially since the max_retries circuit breaker already is enabled by default to a low value. This change will certainly result in a dramatic increase in retry overflows when circuit breaking occurs.

Reply:
As an aside, I really wonder if we should at some point turn on the budgets by default vs. the hard limit. It's a much better default?

Reply:
@tonya11en  I will adjust the doc.

@mattklein123 Agreed. Retry budgets are better defaults than hard limits. We should consider doing that